### PR TITLE
Albert customizations

### PR DESCRIPTION
**Pull request by raikkon88**

### CODE REVIEW

File: config.sh [hunk 1]
```diff
@@ -51,23 +51,27 @@ declare -a APT_GET_APPS=(
 )
 
 declare -a BREW_CASK_APPS=(
```
Comment (albertvila) [inline config.sh:53]: Maybe this BREW_CASK_APPS change should be moved to your forked repository? I agree that maybe not everyone is going to use alfred, but some decisions are very subjective. Not sure, maybe I need to rethink this a little bit

File: lib/os.sh [hunk 2]
```diff
@@ -109,19 +109,8 @@ function _setup_osx() {
   # Setting default language English
   defaults write NSGlobalDomain AppleLocale en_ES
 
-  # Setting iterm custom folder config
-  sed -i.bak "s;/Users/albert/workspace/dotfiles;${DOTFILES_DIR};g" "${DOTFILES_DIR}"/iterm/com.googlecode.iterm2.plist
-  defaults write com.googlecode.iterm2 "PrefsCustomFolder" -string "${DOTFILES_DIR}/iterm"
```
Comment (albertvila) [inline lib/os.sh:114]: Why those lines of code are removed?

File: lib/os.sh [hunk 3]
```diff
@@ -109,19 +109,8 @@ function _setup_osx() {
...
-  defaults write com.googlecode.iterm2 "LoadPrefsFromCustomFolder" -bool true
-  ok "iTerm's custom config folder set"
-
   killall Dock Finder SystemUIServer
 
-  # Install the Solarized Dark theme for iTerm just the first time
-  if is_first_execution; then
-    open "${DOTFILES_DIR}/iterm/themes/Solarized Dark.itermcolors"
```
Comment (albertvila) [inline lib/os.sh:122]: Are u saying this is no longer needed?

File: zsh/zshrc [hunk 4]
```diff
@@ -106,3 +106,5 @@ export PATH="$HOME/.jbang/bin:$PATH"
 export PYENV_ROOT="$HOME/.pyenv"
 command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
 eval "$(pyenv init -)"
+
+load-nvm
```
Comment (albertvila) [inline zsh/zshrc:110]: I think I removed this line for performance issues. With it, the zsh startup time is increased by 0.5s, you can check it with the following command: 

```
for i in {1..10}; do /usr/bin/time zsh -i -c exit; done
```

Right now the nvm is automatically loaded when you do a cd to a folder than contains a .nvmrc file.

Not sure if there is another alternative

**Comment by raikkon88** [comment]
This was my first time using a fork and it's not that easy to sync changes from my repo to your repo, I think I will create a branch and open a pull request directly in your repository so all the stuff comming from my personal configuration is not present and I can just suggest some fixes. 

Sorry for the inconvenience, I'll close this.

**Pull request closed without merging**